ModulemdTranslationHelpers/Utils.py

```diff
@@ -1,16 +1,100 @@
 import sys
+import logging
+import gi
 from babel.messages import Catalog, pofile
 from datetime import datetime
 from collections import defaultdict
 
 try:
```

Comment:
This whole `try`/`except` block should be removed, actually. The `sys.exit(77)` thing is only intended for tests. If the translation helpers don't have python-libmodulemd, it should just outright fail.